addons/html_builder/static/src/core/building_blocks/builder_select_item.xml

```diff
@@ -3,27 +3,34 @@
 
 <t t-name="html_builder.BuilderSelectItem">
     <BuilderComponent>
-        <div
+        <div class="d-flex o-hb-select-item-wrapper">
+            <div
             t-attf-class="o-hb-select-dropdown-item d-flex flex-column cursor-pointer o-dropdown-item dropdown-item o-navigable #{ props.className }"
-            t-att-class="{'active': this.state.isActive}"
-            t-att-data-action-id="info.actionId"
-            t-att-data-action-param="info.actionParam"
-            t-att-data-action-value="info.actionValue"
-            t-att-data-class-action="info.classAction"
-            t-att-data-style-action="info.styleAction"
-            t-att-data-style-action-value="info.styleActionValue"
-            t-att-data-attribute-action="info.attributeAction"
-            t-att-data-attribute-action-value="info.attributeActionValue"
-            t-att-title="props.title"
-            t-on-click="this.onClick"
-            t-on-pointerenter="this.onPointerEnter"
-            t-on-pointerleave="this.onPointerLeave"
-            t-on-focusin="() => this.onFocusin()"
-            t-on-focusout="() => this.onFocusout()"
-            t-ref="item"
-            role="menuitem"
-            tabindex="0">
-            <t t-slot="default" />
+                t-att-class="{'active': this.state.isActive}"
+                t-att-data-action-id="info.actionId"
+                t-att-data-action-param="info.actionParam"
+                t-att-data-action-value="info.actionValue"
+                t-att-data-class-action="info.classAction"
+                t-att-data-style-action="info.styleAction"
+                t-att-data-style-action-value="info.styleActionValue"
+                t-att-data-attribute-action="info.attributeAction"
+                t-att-data-attribute-action-value="info.attributeActionValue"
+                t-att-title="props.title"
+                t-on-click="this.onClick"
+                t-on-pointerenter="this.onPointerEnter"
+                t-on-pointerleave="this.onPointerLeave"
+                t-on-focusin="() => this.onFocusin()"
+                t-on-focusout="() => this.onFocusout()"
+                t-ref="item"
+                role="menuitem"
+                tabindex="0">
+                <t t-slot="default" />
+            </div>
+            <t t-if="props.buttonOptions">
+                <button t-on-click="props.buttonOptions.onClick"
+                        t-attf-class="fa #{props.buttonOptions.icon} btn o-hb-select-item-button o-navigable"
+                        t-att-title="props.buttonOptions.title" />
```

Comment:
Why not add it when you use `BuilderSelectItem`?
<BuilderSelectItem>Blabla <button/></BuilderSelectItem>